src/pages/RepoPage/FailedTestsTab/FailedTestsPage/SelectorSection/BranchSelector/BranchSelector.tsx

```diff
@@ -89,7 +89,17 @@ const BranchSelector = () => {
           // @ts-expect-error - Select has some TS issues because it's still written in JS
           dataMarketing="branch-selector-test-results-tab"
           ariaName="test results branch selector"
-          items={branchList?.branches ?? []}
+          items={
+            overview?.defaultBranch
```

Comment:
I think it might be cleaner if we define a function for this logic and just do items={getBranchItems()}. Up to you

Reply:
I can get behind that, I also wasn't a huge fan of stuffing it in. Will make the update before merging

Reply:
Fixed in [6c8a8ce](https://github.com/codecov/gazebo/pull/3477/commits/6c8a8ce26f06a20f1cce41cac9e24e4e3e38f759)